### PR TITLE
Fix raytrace bug 

### LINKED ISSUE #28: RayTracer returns incorrect entities when there is no hit
When using the RayTracer whilst not correctly pointing at a furniture entity, it (sometimes?) returns the wrong value. F.e.:
```[INFO][/action_server][1580567041.468716]: Received rgb, depth, cam_info
[INFO][/action_server][1580567042.126532]: I see an operator at 2.07 meter in front of me
[INFO][/action_server][1580567043.157913]: There is a ray intersection with trashbin at (3.541, 5.5, 0.8894)
[INFO][/action_server][1580567043.159017]: 'You pointed at trashbin'
```
This is incorrect since the trashbin is at ```(1.32, 5.3, 0.0)```

### PR DESCRIPTION
**Pull request by JosjaG**
Fixes #28

### CODE REVIEW

File: ed_sensor_integration/src/kinect/ray_tracer.cpp [hunk 1]
```diff
@@ -19,16 +19,21 @@ class PointRenderResult : public geo::LaserRangeFinder::RenderResult
...
-    PointRenderResult() : geo::LaserRangeFinder::RenderResult(dummy_ranges_) {}
+    PointRenderResult() : geo::LaserRangeFinder::RenderResult(dummy_ranges_), depth_(0.0), entity_("") {}
 
     void renderPoint(int index, float depth)
     {
         float old_depth = depth_;
-        if (old_depth == 0 || depth < old_depth)
+        if (old_depth == 0.0 || depth < old_depth)
         {
+            ROS_DEBUG_STREAM("Using " << active_entity_ << " as active entity, depth from " << old_depth << " to " << depth);
```
Comment (MatthijsBurgh) [inline ed_sensor_integration/src/kinect/ray_tracer.cpp:29]: Not sure about the performance influence of this.

File: ed_sensor_integration/src/kinect/ray_tracer.cpp [hunk 2]
```diff
@@ -19,16 +19,21 @@ class PointRenderResult : public geo::LaserRangeFinder::RenderResult
...
             depth_ = depth;
             entity_ = active_entity_;
         }
+        else
+        {
+            ROS_DEBUG_STREAM("Not using entity " << active_entity_);
```
Comment (MatthijsBurgh) [inline ed_sensor_integration/src/kinect/ray_tracer.cpp:35]: Not sure about the performance influence of this.

File: ed_sensor_integration/src/kinect/ray_tracer.cpp (hunk 1 above)
Comment (jlunenburg) [reply ed_sensor_integration/src/kinect/ray_tracer.cpp:29]: AFAIK, this is *only* called if the ray actually hits an entity (or volume), so usually only a couple of times per raytrace call. I therefore don't expect this to have a significant influence on the performance.

File: ed_sensor_integration/src/kinect/ray_tracer.cpp (hunk 2 above)
Comment (jlunenburg) [reply ed_sensor_integration/src/kinect/ray_tracer.cpp:35]: Idem

**Review by MatthijsBurgh — APPROVED** [review]

**Pull request merged**